bin/assets/scripts/newform-button.js

```diff
@@ -0,0 +1,20 @@
+const code = document.getElementsByName("code")[0];
+const button = document.querySelector(".post-snippet button");
+const svg = button.querySelector("svg");
```

Comment:
The element are not known yet, to be move in a function `window.addEventListener("load", function () {...});`

Reply:
Actually they are, as the script is called with `defer` attribute, which loads it *after* it loads the HTML page

Reply:
oh great ! didn't know about that attribute !